mce-io.c

```diff
@@ -24,6 +24,7 @@
 #include <errno.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
```

Comment:
Duplicate of https://github.com/maemo-leste/mce/pull/3/commits/157b5615ce5edfb00115f2ccc5fc2a7ce0f8f919

Reply:
Ok, will drop this commit.